### PR TITLE
created a footer - needs to review

### PR DESCRIPTION
**Pull request by andsemenov**

### CODE REVIEW

File: css/footer.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  height: 60px;
+  line-height: 60px;
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 40px;
+}
+
+.back-to-top a {
+  color: #000000;
+  font-family: Poppins;
```
Comment (Lily-La-Day) [inline css/footer.css:40]: @andsemenov Everything here looks great but when I run this locally I don't see this font
Screenshot 2021-03-02 at 10 23 22

Comment (andsemenov) [reply css/footer.css:40]: Thanks for this. I added link to font family, it works for me. Could you check it again, please?

**Review by Lily-La-Day — APPROVED** [review]
Excellent work! ⭐

File: css/footer.css (hunk 1 above)
Comment (Lily-La-Day) [reply css/footer.css:40]: yes, all good now and looking just like the design - thank you!

**Review by mohamedlafiteh — APPROVED** [review]
It looks great. Well done.

**Pull request closed without merging**